common/src/main/java/org/dash/wallet/common/services/analytics/AnalyticsConstants.kt

```diff
@@ -1,17 +1,20 @@
 /*
- * Copyright 2021 Dash Core Group
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ *  * Copyright 2021 Dash Core Group.
```

Comment:
There are a few license headers that were indented.

`*  *`

Reply:
I just copy-pasted this one from some other file. Will fix.